test/large-app/app.e2e-spec.ts

```diff
@@ -37,3 +41,13 @@ SpelunkerSuite('Should allow the SpelunkerModule to debug', async () => {
   const output = await SpelunkerModule.debug(AppModule);
   equal(output, debugOutput);
 });
+
+SpelunkerSuite('Should allow the SpelunkerModule to graph', ({ app }) => {
+  const tree = SpelunkerModule.explore(app);
+  const root = SpelunkerModule.graph(tree);
+  const edges = SpelunkerModule.findGraphEdges(root);
+  equal(
+    edges.map((e) => `${e.from.module.name}-->${e.to.module.name}`),
```

Comment:
fwiw: this is the core of the mermaid syntax, but thats not really the point of findGraphEdges. The SpelunkedNode and SpelunkedEdge data structures can be easily projected to whatever syntax you want.